src/cg/cst_cg_dump_voice.c

```diff
@@ -88,13 +88,13 @@ static void cst_cg_write_tree_nodes(cst_file fd, const cst_cart_node * nodes)
     while (nodes[num_nodes].val != 0)
         num_nodes++;
 
-    cst_fwrite(fd, &num_nodes, sizeof(int), 1);
+    cst_fwrite(fd, &num_nodes, sizeof(int32_t), 1);
```

Comment:
`num_nodes` should probably be declared as `int32_t`. Probably not a real issue but it might be nice for consistency. Ints are mostly 32-bit